@AH-64D Apache Longbow/Addons/fza_ah64_hatchet/includes/script_macros_common.hpp

```diff
@@ -0,0 +1,1857 @@
+/*
```

Comment:
If this is due to scons not liking it, I built an enhanced version of scons you can use for the hatchet team, that allows for an include for
https://github.com/Project-Hatchet/H-60/tree/Main/include